src/main/java/edu/kis/powp/jobs2d/drivers/MacroDriver.java

```diff
@@ -8,34 +8,36 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class MacroDriverDecorator implements Job2dDriver {
+public class MacroDriver implements Job2dDriver {
     private List<DriverCommand> driverCommandList = new ArrayList<>();
-    private Job2dDriver job2dDriver;
+    private DriverComposite driverComposite;
```

Comment:
No, macro driver should not contain a driver composite! Macro should only perform macro recording funcion (SRP principle)